doc/manual/manual.tex

```diff
@@ -10432,6 +10432,89 @@ \subsubsection{Donea \& Huerta 2D box geometry benchmark}
+\sigma_P(T^{\prime}) = \begin{cases}
+4  &  \text{for }T^{\prime} < 0.92 \\
+4+37.5(T^{\prime}-0.92) & \text{for }0.92\leq T^{\prime} < 1 \\
+7& \text{for } T^{\prime} \geq 1 \\
+\end{cases}
+\end{equation}
+where $T^{\prime}$ is homologous temperature ($\frac{T}{T_{s}}$) with $T$ the temperature and $T_s$ the solidus temperature, both in Kelvin. $\phi_m$ is the melt fraction and $\beta(\phi_m)$ describes the direct poroelastic effect of melt (assumed to be negligible under upper mantle conditions). For this case, $J_U$ is the inverse of the unrelaxed shear modulus, $\mu_{U}(P,T)$, such that
+\begin{equation}
+J_{U}(P,T)^{-1} = \mu_U(P,T) = \mu_U^0 + \frac{\partial{\mu_U}}{\partial{T}}(T -T_0)+ \frac{\partial{\mu_U}}{\partial{P}}(P-P_0)
+\end{equation}
+where $\mu_U^0$ is the unrelaxed shear modulus at surface pressure-temperature conditions, the differential terms are assumed to be constant and the pressure, $P$, in GPa is linearly related to the depth, $z$, in km by $\frac{z}{30}$. The normalised shear wave period, $\tau_S^{\prime}$, in Equations~(\ref{eq:real_compliance}) and (\ref{eq:imaginary_compliance}) is equal to $\frac{\tau_S}{2\pi\tau_M}$, where $\tau_S = 100$~s is the shear wave period and $\tau_M = \frac{\eta}{\mu_U}$ is the normalised Maxwell relaxation timescale. $\tau_P^{\prime}$ represents the normalised shear-wave period associated with the centre of the high frequency relaxation peak, assumed to be $6 \times 10^{-5}$. The shear viscosity, $\eta$, is
+\begin{equation}
+\eta = \eta_r \left(\frac{d}{d_r}\right)^{m} \text{exp} \left[ \frac{E_a}{R}\left(\frac{1}{T}-\frac{1}{T_r}\right) \right] \text{exp} \left[ \frac{V_a}{R}\left(\frac{P}{T}-\frac{P_r}{T_r}\right) \right] A_{\eta}
+\label{eq:eta}
+\end{equation}
+where $d$ is the grain size, $m$ the grain size exponent (assumed to be 3), $R$ the gas constant, $E_a$ the activation energy and $V_a$ the activation volume. Subscripts $[X]_r$ refer to reference values, assumed to be $d_r = d = 1$~mm, $P_r = 1.5$~GPa and $T_r = 1200^{\circ}$C for the upper mantle. $A_{\eta}$ represents the extra reduction of viscosity due to an increase in $E_a$ near the solidus, expressed as
+\begin{equation}
+A_\eta(T^{\prime}) = 
+\begin{cases}
+1  & \text{for } T^{\prime} < T^{\prime}_{\eta} \\
+\text{exp} \left[-\frac{(T^{\prime}-T^{\prime}_{\eta})}{(T^{\prime}-T^{\prime}T^{\prime}_{\eta})} \text{ln}(\gamma)\right]& \text{for }T^{\prime}_{\eta} \leq T^{\prime} < 1 \\
+\gamma^{-1} \text{exp}(\lambda\phi) & \text{for }T^{\prime} \geq 1\\
+\end{cases}
+\end{equation}
+where $T^{\prime}_\eta$ is the homologous temperature above which activation energy becomes $E_a + \Delta E_a$ and $\gamma = 5$ is the factor of additional reduction. $\lambda\phi$ describes the direct effect of melt on viscosity, assumed to be negligible here. The solidus temperature, $T_s$, is fixed to a value of 1599 K at 50~km equivalent to a dry peridotite solidus \cite{Hirsch2000} and linearly increases below this depth according to
```

Comment:
, which ?

Reply:
"above which" is grammatically correct but I have put a comma before "and $\gamma..." so it's clear that the statements about activation energy and $\gamma$ are separate.